addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -280,6 +282,7 @@ export const dateField = {
         rounding: options.rounding && parseInt(options.rounding, 10),
         startDateField: options[START_DATE_FIELD_OPTION],
         warnFuture: archParseBoolean(options.warn_future),
+        forceRange: options.force_range || dynamicInfo.required,
```

Comment:
Why not use `forceRange: options.force_range` and then we can still check `this.props.forceRange || this.props.required || ...` ?

Reply:
Because `required == True` implies a range is required regardless, the `force_range` option is just a way to do that without making the date fields themselves required.